### DISCUSSION TITLE
randomly renumbering problem

### DISCUSSION THREAD

**Discussion by DalecB** [nftchef/art-engine#119] [Help + Q&A] [answered]
HI @nftchef , I always appreciate on your work!

I wonder if I can mix two different collections in to one big collection and renumber it randomly.
And update its metadata without touching any other features except name 
(eg. **312.json** / **312.png** / **Series 2 # 312** _-(renumbering(random))->_ **25.json** / **25.png** / **Series 2 # 25**)
from 1 to 10K

**Comment by nftchef** [comment, marked as answer]
Thank you!

There's no built-in way to do this, but it wouldn't be too hard to write a script to read in the files from both folders (projects) and mix them together, and rewrite the metadata.

**Reply by DalecB** [reply]
thanks for ur reply! it helped me alot
